apps/antalmanac/src/routes/Error.tsx

```diff
@@ -0,0 +1,48 @@
+import { Typography, Button, Stack } from '@mui/material';
+import { useRouteError } from 'react-router-dom';
+import { Link } from 'react-router-dom';
```

Comment:
```suggestion
import { Link, useRouteError } from 'react-router-dom';
```
nit: no dupe imports